project_recalculate/README.rst

```diff
@@ -0,0 +1,100 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+===================
+Project Recalculate
+===================
+
+This module recalculates tasks' start/end dates depending on project's
+start/end dates.
+
+
+Instalation
+===========
+
+This addon requires Odoo v8 after 2015-08-24 because is not fully functional
+without this patch: `Odoo PR #8208 <https://github.com/odoo/odoo/pull/8208>`
+
+
+Configuration
+=============
+
+You can define working calendar at Setting > Technical > Resource > Working time
+Then assign this calendar to a resource (related with an user) or a project
+
+When calculating task dates, this addon will look for a working calendar in this order:
+
+* If project has working time assigned, use it.
+* If user assigned, search first resource related with this user
+  (normally user is related, only with one resource) and get its working calendar
+  ends at 18:00
+* If not user assigned or resource hasn't working calendar, search first
+  working calendar of the company
+* If no working calendar found, then every day is workable and work starts at
+  08:00 and ends at 18:00
+
+
+Usage
+=====
+
+There are two calcutation modes:
+
+* **Date begin**: Task start/end dates are recalculated from project's date begin
+* **Date end**: Task start/end dates are recalculated from project's date end
+
+If the project start/end date is changed in the form view, then you have to
+click "Recalculate project" button to recalculate all open tasks [1]
+according to the new date.
+
+[1] 'Open tasks' means tasks in a stage that are defined with 'fold' = False
+
+This a typical use case:
+
+1. Create a project and configure:
+    * Calculation type, for example "Date end"
+    * Date end
+2. Create tasks, configuring for each one:
+    * From days, in this example, days from date end when this task must start
```

Comment:
"Estimated days" (```estimated_days``` field) means "Duration Days".
"From Days" are the days from project begin date (or end date, depeding on calculation type).